CentralComputing/I2CManager.cpp

```diff
@@ -98,24 +98,38 @@ bool I2CManager::single_shot(int fd, int port, int16_t *value) {
 
 std::shared_ptr<I2CData> I2CManager::refresh() {
   std::shared_ptr<I2CData> new_data = std::make_shared<I2CData>();
-  new_data->dummy_data = i;
-  i++;
+  if (i == 3) {
+    i = 0;
+  } else {
+    i++;
+  }
+  int port;
+  int addr;
+  if (i == 0) {
+    port = ANC0;
+    addr = 0x48;
```

Comment:
Please note my comment about the addresses.

Reply:
I hardcoded it in for now. Want to get a test for the I2C up and running and focus on some priorities first. If we are still in good shape I can revisit making it more configurable